src/UserSettingsPage.cpp

```diff
@@ -2127,11 +2127,30 @@ UserSettingsModel::exportSessionKeys()
     if (!ok)
         return;
 
+    auto repeatedPassword = QInputDialog::getText(nullptr,
+                                                  tr("Repeat File Password"),
+                                                  tr("Repeat the passphrase:"),
+                                                  QLineEdit::Password,
+                                                  QLatin1String(""),
+                                                  &ok);
+    if (!ok)
+        return;
+
     if (password.isEmpty()) {
         QMessageBox::warning(nullptr, tr("Error"), tr("The password cannot be empty"));
         return;
     }
```

Comment:
Can you move this check up to behind the first dialog? There is no need to ask the user for a second password, if the first one is invalid.